UPGRADING.md

```diff
@@ -0,0 +1,182 @@
+# 3.0.0 Migration Guide
+
+The 3.0 release of the `google-cloud-translate` client is a significant upgrade based on a [next-gen code generator](https://github.com/googleapis/gapic-generator-python), and includes substantial interface changes. Existing code written for earlier versions of this library will likely require updates to use this version. This document describes the changes that have been made, and what you need to do to update your usage.
+
+If you experience issues or have questions, please file an [issue](https://github.com/googleapis/python-translate/issues).
+
+## Supported Python Versions
+
+> **WARNING**: Breaking change
+
+The 3.0.0 release requires Python 3.6+.
+
+
+## Method Calls
+
+> **WARNING**: Breaking change
+
+Methods expect request objects. We provide a script that will convert most common use cases.
+
+* Install the library
+
+```py
+python3 -m pip install google-cloud-translate
+```
+
+* The script `fixup_translation_{version}_keywords.py` is shipped with the library. It expects
+an input directory (with the code to convert) and an empty destination directory.
+
+```sh
+$ fixup_translation_v3_keywords.py --input-directory .samples/ --output-directory samples/
+```
+
+**Before:**
+```py
+from google.cloud import translate
+
+client = translate.TranslationServiceClient()
+parent = client.location_path(<PROJECT_ID>, <LOCATION>)
+
+response = client.translate_text(
+    parent=parent,
+    contents=[text],
```

Comment:
Could you declare the variable and set it to some value before the call?

```py
text = "Good morning!"

response = client.translate_text(...

```

Reply:
Done!